fe/fe-core/src/main/java/com/starrocks/common/PriorityFutureTask.java

```diff
@@ -0,0 +1,41 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.common;
+
+import java.util.concurrent.FutureTask;
+
+public class PriorityFutureTask<V> extends FutureTask<V> implements
+        Comparable<PriorityFutureTask<V>> {
+
+    private PriorityRunnable run;
+
+    public PriorityFutureTask(PriorityRunnable runnable, V result) {
+        super(runnable, result);
+        this.run = runnable;
+    }
+
+    public int getPriority() {
+        return run.getPriority();
+    }
+
+    public void setPriority(int priority) {
```

Comment:
Use High/Middle/Low to replace the 1/2/3

Reply:
why only support three level?

Reply:
How many priorities do you want to provide?